server/controllers/apply/applicationsController.test.ts

```diff
@@ -424,6 +424,7 @@ describe('applicationsController', () => {
           errors: errorsAndUserInput.errors,
           errorSummary: errorsAndUserInput.errorSummary,
           ...errorsAndUserInput.userInput,
+          restrictedPerson: person.crn,
```

Comment:
This value is set to be 'true' in People Controller but here we are setting it to be a CRN

Reply:
updated